src/main/resources/templates/pps/bankTransferPenaltyReference.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+
+<html xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layouts/baseLayout}">
+
+<head>
+  <title>
+    What type of penalty do you need to pay?
+  </title>
+</head>
+
+<div id="pps-penalty-reference"  layout:fragment="content">
+  <form id="ppspenaltyreference" th:action="@{''}" th:object="${penaltyReferences}" method="post" class="form">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <div th:replace="fragments/ppsGlobalErrors :: globalErrors"></div>
+
+        <h1 id="page-title " class="govuk-heading-xl">
+          <span id="page-title-heading">Find out which bank details to use</span>
+        </h1>
+        <p class="govuk-body">We need to ask you about your penalty reference, so that we can show you the correct bank details for the type of penalty you’re paying.</p>
+        <h2 class="govuk-heading-m">What does the penalty reference start with?</h2>
+        <p class="govuk-hint govuk-!-margin-top-3">For example, the penalty reference A1234567 starts with A.</p>
+        <div class="govuk-form-group" th:classappend="${#fields.hasErrors('selectedPenaltyReference')} ? 'govuk-form-group--error' : ''">
+          <fieldset class="govuk-fieldset">
+            <span class="govuk-error-message"
+                  id="selectedPaymentMethod-errorId"
+                  th:if="${#fields.hasErrors('selectedPenaltyReference')}"
+                  th:each="e : ${#fields.errors('selectedPenaltyReference')}" th:text="${e}">
+            </span>
+
+            <div class="govuk-radios" th:each="penaltyReference : ${availablePenaltyReference}" style="margin-bottom:1em;">
```

Comment:
I think we might need to add in `data-module="govuk-radios"` here

Reply:
Updated from the review suggested